src/Symfony/Component/PropertyInfo/Extractor/SerializerExtractor.php

```diff
@@ -48,11 +50,18 @@ public function getProperties(string $class, array $context = []): ?array
 
         foreach ($serializerClassMetadata->getAttributesMetadata() as $serializerAttributeMetadata) {
             $ignored = method_exists($serializerAttributeMetadata, 'isIgnored') && $serializerAttributeMetadata->isIgnored();
-            if (!$ignored && (null === $context['serializer_groups'] || array_intersect($context['serializer_groups'], $serializerAttributeMetadata->getGroups()))) {
+            if (!$ignored && (null === $context['serializer_groups'] || array_intersect($context['serializer_groups'], $this->getAttributeGroups($serializerAttributeMetadata)))) {
                 $properties[] = $serializerAttributeMetadata->getName();
             }
         }
 
         return $properties;
     }
+
+    private function getAttributeGroups(AttributeMetadataInterface $serializerAttributeMetadata): array
+    {
+        $groups = empty($serializerAttributeMetadata->getGroups()) ? [AbstractNormalizer::DEFAULT_GROUP_FOR_ATTRIBUTE_WITHOUT_GROUPS] : $serializerAttributeMetadata->getGroups();
+
+        return array_merge($groups, ['*']);
```

Comment:
```suggestion
        $groups[] = '*';

        return $groups;
```

This will also probably be a bit faster.